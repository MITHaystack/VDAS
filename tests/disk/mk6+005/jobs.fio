[disk8]
rw=write
iodepth=8
directory=/mnt/disk8
write_bw_log=disk8_write
fill_device=1

[disk9]
rw=write
iodepth=8
directory=/mnt/disk9
write_bw_log=disk9_write
fill_device=1

[disk10]
rw=write
iodepth=8
directory=/mnt/disk10
write_bw_log=disk10_write
fill_device=1

[disk11]
rw=write
iodepth=8
directory=/mnt/disk11
write_bw_log=disk11_write
fill_device=1

[disk12]
rw=write
iodepth=8
directory=/mnt/disk12
write_bw_log=disk12_write
fill_device=1
stonewall

[disk13]
rw=write
iodepth=8
directory=/mnt/disk13
write_bw_log=disk13_write
fill_device=1

[disk14]
rw=write
iodepth=8
directory=/mnt/disk14
write_bw_log=disk14_write
fill_device=1

[disk15]
rw=write
iodepth=8
directory=/mnt/disk15
write_bw_log=disk15_write
fill_device=1
