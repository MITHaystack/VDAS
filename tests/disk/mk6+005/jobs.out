disk8;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961351280;104451;9424716;0;0;0.000000;0.000000;2;468761;38.126571;1430.191297;20145;1082584;26.663254%;102529.543763;24177.401865;3.126142%;12.963575%;491957;0;424;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;21.10%;77.00%;0.41%;0.33%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

disk9;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961351388;104231;9444624;0;0;0.000000;0.000000;2;1693797;38.190203;1432.577078;22743;1075480;26.615922%;102347.535435;24469.319736;3.196634%;12.986966%;492875;0;419;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;20.99%;76.99%;0.54%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.02%;0.01%;0.01%;0.00%;0.00%;0.01%;0.00%

disk10;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;958541772;98225;9992819;0;0;0.000000;0.000000;2;1394820;40.611549;1703.671233;5030;1060536;25.171368%;96792.719046;27532.222397;2.956523%;12.120705%;487947;0;432;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;22.26%;75.81%;0.44%;0.35%;0.99%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.02%;0.01%;0.01%;0.01%;0.01%;0.01%;0.00%

disk11;0;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961349668;107118;9190009;0;0;0.000000;0.000000;2;926243;37.143854;1383.388602;17543;1057600;27.346988%;105158.738387;25450.741425;3.197386%;13.297050%;486924;0;407;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;21.42%;76.67%;0.44%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.02%;0.01%;0.01%;0.00%;0.01%;0.00%;0.00%

disk12;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;958541808;100765;9740872;0;0;0.000000;0.000000;2;1272299;39.559634;1483.862389;2949;293104;25.078234%;98928.867638;22890.034218;3.061225%;12.535019%;524327;0;407;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;20.74%;77.33%;0.45%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.01%;0.00%;0.01%;0.00%

disk13;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961349720;106913;9207636;0;0;0.000000;0.000000;2;543168;37.222105;1376.527949;29257;367171;26.597656%;104922.700839;23063.427898;3.225693%;13.232171%;517213;0;393;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;20.97%;77.17%;0.38%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.02%;0.01%;0.01%;0.01%;0.00%;0.00%;0.00%

disk14;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961350032;106461;9246781;0;0;0.000000;0.000000;2;408817;37.389536;1393.458783;28230;262144;26.482753%;104469.427735;23961.073972;3.194409%;13.154741%;517967;0;394;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;21.20%;76.92%;0.40%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.02%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

disk15;1;0;0;0;0;0;0;0.000000;0.000000;0;0;0.000000;0.000000;0;0;0.000000%;0.000000;0.000000;961352040;103290;9530635;0;0;0.000000;0.000000;2;446810;38.576552;1426.627800;37022;216992;25.689391%;101339.765966;22299.409191;3.081852%;12.791174%;526834;0;402;100.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.0%;0.00%;21.10%;77.04%;0.37%;0.34%;1.00%;0.06%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.01%;0.04%;0.03%;0.01%;0.01%;0.00%;0.00%;0.00%;0.00%

